src/components/shared/ManagedTree.js

```diff
@@ -6,7 +6,7 @@
 import React, { Component } from "react";
 import "./ManagedTree.css";
 
-import { Tree } from "devtools-components";
+const { Tree } = require("devtools-components");
```

Comment:
I couldn't get this to resolve correctly, so I switched it to a CommonJS require to opt out of eslint import.